ivadomed/loader/loader.py

```diff
@@ -69,21 +69,6 @@ def load_dataset(bids_df, data_list, transforms_params, model_params, target_suf
                                 object_detection_params=object_detection_params,
                                 soft_gt=soft_gt,
                                 is_input_dropout=is_input_dropout)
-    # elif model_params[ModelParamsKW.NAME] == ConfigKW.HEMIS_UNET:
-    #     dataset = imed_adaptative.HDF5Dataset(bids_df=bids_df,
-    #                                           subject_file_lst=data_list,
-    #                                           model_params=model_params,
-    #                                           contrast_params=contrast_params,
-    #                                           target_suffix=target_suffix,
-    #                                           slice_axis=imed_utils.AXIS_DCT[slice_axis],
-    #                                           transform=tranform_lst,
-    #                                           metadata_choice=metadata_type,
-    #                                           slice_filter_fn=SliceFilter(**slice_filter_params,
-    #                                                                                         device=device,
-    #                                                                                         cuda_available=cuda_available),
-    #                                           roi_params=roi_params,
-    #                                           object_detection_params=object_detection_params,
-    #                                           soft_gt=soft_gt)
```

Comment:
I'm kinda inclined towards not changing this partly bcz I lack knowledge about HEMIS_UNET and the related functionality, but there might definitely be a reason for using HDF5Dataset that might help us in giving a lead to deal with large datasets.

Reply:
So, what happened was a while back, HEMIS_UNET was deprecated by us, so that was why it was commented out to be eventually removed.